### ISSUE TITLE
class "Player" : when pausing the player, and then calling the seek() method, it makes it unpause

### ISSUE THREAD

**Issue by Romuald78** [pythonarcade/arcade#828]
## Bug Report
It seems there is an issue in the class "Player" hwen using the seek() method while in pause.

### Actual behavior:
When creating a Sound object, and playing it, a Player instance is created.
Calling the pause() method from it works correctly. the sound is not played anymore.
Then calling the Player.seek() method with any value inside, makes the sound plays again (unpause).

Additional info : Calling the Player.play() method again after a pause() method call resumes the sound correctly. 

### Expected behavior:
When just calling the  Player.seek() method, the isPlaying local variable contains the correct value (False) so i would except the sound not to play.

### Steps to reproduce/example code:

**In the application init() method :** 
```
        self._music = arcade.load_sound("resources/sfx/music/test001.mp3")
        self._player = self._music.play()
        self._player.pause()
```

**In the keyboard event method :** 
```
        if key == arcade.key.SPACE and isPressed:
            if self._player.playing:
                self._player.pause()
            else:
                self._player.play()

        if key == arcade.key.BACKSPACE and isPressed:
            self._player.seek(0)
```

I wanted to control the playing operation from the player object instead of from the sound object because i didn't want to RE-create a new player each time i pause (because the Sound.stop() method calls the Player.delete() method and thought it was useless to delete / create new players each time we pause/play and updating the media.Source_players list too

Even when adding a Player.pause() method after, it unpauses the sound
```
        if key == arcade.key.SPACE and isPressed:
            if self._player.playing:
                self._player.pause()
            else:
                self._player.play()

        if key == arcade.key.BACKSPACE and isPressed:
            self._player.seek(0)
            self._player.pause()
```

### What would it help with?

For the moment I keep in memory a copy of the seek time and only call the Player.seek() method just before really playing the sound

**Comment by Cleptomania** [comment]
Sorry for the wait on replying to this, I'm unable to reproduce the issue of the sound automatically playing after using seek, either through Arcade or directly in Pyglet. However I do notice some weird behavior where if you seek to the beginning and then pause, when you unpause it will begin playing from where it was before seeking, and then seek to the beginning, I'm not 100% sure what's going wrong there.

This difference in behavior leads me to believe they could be issues with the specific audio backend being used and you and I are getting different backends due to OS configuration, but I don't have the time look into this too deeply right now.

However, I think you may be overestimating how heavy of a lift creating/destroying the players is. I'll assume for this it's playing music of some sort? I am certain you will not run into any performance issues with using `Sound.stop()` and then playing the sound again. The only real heavy lifting from the sound module comes from loading the sounds, if you were having to do that over and over, it would be more of a problem.

I've included a template with this behavior below.

```python
import arcade

SCREEN_WIDTH = 800
SCREEN_HEIGHT = 600
SCREEN_TITLE = "Sound Template"


class MyGame(arcade.Window):

    def __init__(self, width, height, title):
        super().__init__(width, height, title)

        arcade.set_background_color(arcade.color.AMAZON)

        self._music = arcade.load_sound("music.mp3")
        self._player = self._music.play()
        # If you have sprite lists, you should create them here,
        # and set them to None

    def on_draw(self):
        arcade.start_render()

    def on_key_press(self, key, key_modifiers):
        if key == arcade.key.SPACE:
            if self._player:
                self._music.stop(self._player)
                self._player = None
            else:
                self._player = self._music.play()


def main():
    game = MyGame(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
    arcade.run()


if __name__ == "__main__":
    main()
```

Let me know if you had any more findings or thoughts on it. I had done tests in which I created upwards of a few hundred players per second when I built out the new sound module and didn't have any issues with it, so I don't think you should have any issue doing it this way.

**Comment by Romuald78** [comment]
Yeah i have tried your code. It does only play and stop (stop in arcade, destroys the player AFAIK) and play creates a new one.
But that does not use the seek() feature so i guess it is not really relevant . Destroying the player, destroys the current playing position.

I agree there must be some issues with the specific hardware or OS things that lead to some different behaviours
to be continued....

**Comment by einarf** [comment]
It would be nice to know what platform you are seeting this issue. Pyglet support several backend for sound across many platforms.

**Comment by pvcraven** [comment]
Closing, reopen if we can get a dev to replicate or more info.

**Comment by Cleptomania** [comment]
Wouldn’t be surprised if 2.6 fixes this, there have been a lot of improvements on Pyglet’s sound system since we implemented it in 2.5
